steps/synopsys-ptpx-genlibdb/scripts/read_design.tcl

```diff
@@ -28,6 +28,18 @@ current_design $ptpx_design_name
 link_design
 
 # Read in the SDC and parasitics
-
-read_sdc -echo $ptpx_sdc
-read_parasitics -format spef $ptpx_spef
+# Try to read the sdc constraints files
+if {[ file exists $ptpx_sdc ]} {
```

Comment:
Super minor, but for coding convention can we add an empty line between the comment and the if statement?

Reply:
Done. :)